wicket-experimental/wicket-metrics/src/main/java/org/apache/wicket/metrics/aspects/WicketFilterRequestCycleAspect.java

```diff
@@ -42,6 +44,35 @@
 	 *             might occur while invoking process request
 	 */
 	@Around("execution(* org.apache.wicket.protocol.http.WicketFilter.processRequestCycle(..))")
+	public Object aroundRequestProcessedWithURL(ProceedingJoinPoint joinPoint) throws Throwable
+	{
+		Object[] args = joinPoint.getArgs();
+		if (args.length >= 3)
+		{
+			Object requestAsObject = args[2];
+			if(requestAsObject instanceof HttpServletRequest){
+				HttpServletRequest httpServletRequest = (HttpServletRequest)requestAsObject;
+				if (httpServletRequest != null)
+				{
+					return measureTime("core/application/request/" + httpServletRequest.getRequestURL().toString().replace("/", "_").replace(".", "_").replace(":", ""), joinPoint,
```

Comment:
Split this in several lines/assignments.
Use `String#replace(char, char)` where possible. 
Otherwise prefer Wicket-Util's `Strings#replace(CharSequence, CharSequence, CharSequence)` than `String#replace(CharSequence, CharSequence)` because the latter uses non-compiled Regex,
i.e. it is much slower.